code/game/objects/items.dm

```diff
@@ -37,9 +37,14 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 
 	var/hitsound
 	var/usesound
-	var/throwhitsound
+	///Used when yate into a mob

```

Comment:
Yate? ___YATE?___ If you're going to do this, at least use the proper past tense `yote`.

Reply:
eat - ate
yeet - yate

Reply:
phonetically the same, I've debatted this with people many times so come at me

Reply:
greet - greeted
yeet - yeeted

:^)

Reply:
I feel like eat is closer so go away

Reply:
it's not yeat, though

Reply:
bro its phonetic ok?